DeAceroDLabs/marketplace-frontend
# Feature/dmp 667 styles to notification

## What is done

[[DMP-667] Shopping cart Notification content](https://deacero.atlassian.net/browse/DMP-667)

## How is done

- Added scss, sumprices funciton, and divider to  `NotificationContent` component
- Added vertical type to `Divider` component.

## How to test

1. Run the project via `npm start`.
2. Click on any product.
3. Click `Agregar al carrito` button.
4. This button should navigate to home with and show a  notification with the correct content.
